cinema/serializers.py

```diff
@@ -68,13 +78,53 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("movie_session", "row", "seat")
```

Comment:
`taken_places` should contain only row and seat (as per task); movie session id - we already have it one level above